content/docs/1.20/getting-started.md

```diff
@@ -94,7 +94,6 @@ mkdir -p $GOPATH/src/github.com/jaegertracing
 cd $GOPATH/src/github.com/jaegertracing
 git clone git@github.com:jaegertracing/jaeger.git jaeger
 cd jaeger
-make install
```

Comment:
We can also remove the two commands with GOPATH, since we switched to go modules and the repo can reside in any location.